nvflare/private/fed/client/client_runner.py

```diff
@@ -204,58 +205,42 @@ def _do_process_task(self, task: TaskAssignment, fl_ctx: FLContext, abort_signal
         self.log_debug(fl_ctx, "firing event EventType.BEFORE_TASK_DATA_FILTER")
         self.fire_event(EventType.BEFORE_TASK_DATA_FILTER, fl_ctx)
 
-        # first apply privacy-defined filters
-        scope_object = fl_ctx.get_prop(FLContextKey.SCOPE_OBJECT)
-        filter_list = []
-        if scope_object:
-            assert isinstance(scope_object, Scope)
-            if scope_object.task_data_filters:
-                filter_list.extend(scope_object.task_data_filters)
-
-        task_filter_list = self.task_data_filters.get(task.name)
-        if task_filter_list:
-            filter_list.extend(task_filter_list)
-
-        if filter_list:
-            task_data = task.data
-            for f in filter_list:
-                filter_name = f.__class__.__name__
-                try:
-                    task_data = f.process(task_data, fl_ctx)
-                except UnsafeJobError:
-                    self.log_exception(fl_ctx, f"UnsafeJobError from Task Data Filter {filter_name}")
-                    executor.unsafe = True
-                    fl_ctx.set_job_is_unsafe()
-                    self.run_abort_signal.trigger(True)
-                    return self._reply_and_audit(
-                        reply=make_reply(ReturnCode.UNSAFE_JOB),
-                        ref=server_audit_event_id,
-                        fl_ctx=fl_ctx,
-                        msg=f"submit result: {ReturnCode.UNSAFE_JOB}",
-                    )
-                except Exception as e:
-                    self.log_exception(
-                        fl_ctx, f"Processing error from Task Data Filter {filter_name}: {secure_format_exception(e)}"
-                    )
-                    return self._reply_and_audit(
-                        reply=make_reply(ReturnCode.TASK_DATA_FILTER_ERROR),
-                        ref=server_audit_event_id,
-                        fl_ctx=fl_ctx,
-                        msg=f"submit result: {ReturnCode.TASK_DATA_FILTER_ERROR}",
-                    )
+        task_data = task.data
+        try:
+            filter_name = f"{Scope().task_data_filters=}".split("=")[0].split(".")[-1]
```

Comment:
This line might be hard to read.

Essentially it is trying to get ```filter_name = "task_data_filters"```

Should we just hard-coded it? or use some constants variable in Scope.py?

Reply:
Same as the other one "task_result_filters". Hard-code will likely cause error when variable renaming.

Reply:
changed to use hard coded field name.